index.html

```diff
@@ -263,6 +263,52 @@ <h2>LET'S CONNECT</h2>
           </div>
         </div>
       </section>
+      <!-- Contact Form here -->
+
+      <section class="contactt">
+        <div class="ttry">
+          <form
+            class="forms"
+            action="https://formspree.io/alusinejalloh800@gmail.com"
+            method="POST"
+          >
+            <h2 class="contactme-head">Contact me</h2>
+            <p class="contactme-body">
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+            <input
+              class="form-name"
+              type="text"
+              maxlength="30"
+              placeholder="Full Name"
+              required
+            />
```

Comment:
- [ ] I suggest adding some spaces (Margins or Paddings) between the paragraph and the inputs to match the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) remember, According to the requirements, You must stick to the design as much as possible (e.g. font, colors, images, tests, margins), so kindly fix it 👌 

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/177541157-ea0ff3bc-e588-4cfb-93a1-d1c99ae132e1.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/177541216-97f5e876-c2f4-4038-b67c-6689b36682f3.PNG" width="500" /> |